onboard/nav/avoidances.hpp

```diff
@@ -0,0 +1,39 @@
+#ifndef AVOIDANCES_HPP
+#define AVOIDANCES_HPP
+
+#include "avoidance.hpp"
+
+// This class is the representation of different avoidance algorithms
+enum class AvoidanceType
+{
+    ORIGINAL,
+    UNKNOWN
+};
+
+Avoidance* AvoidFactory( StateMachine* stateMachine, AvoidanceType type );
+
+/*************************************************************************/
+/* Default Avoidance */
```

Comment:
This comment doesn't seem super helpful nor accurate. It is currently our only avoidance algorithm not our default.